presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -271,3 +271,12 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
 .. note::
 
     Unfortunately, there is no way to represent ``_id`` fields more fancy like ``55b151633864d6438c61a9ce``.
+
+Limitations
+-----------
+
+The following operations are not supported:
+
+- Deleting data with :doc:`/sql/delete`, :doc:`/sql/drop-table`, or similar
```

Comment:
DROP TABLE is supported in MongoDB connector. 
https://github.com/prestosql/presto/blob/37c74ad15f5f0c9c770c2aa491e49ffe5146d4b1/presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoMetadata.java#L159-L165

Reply:
oh ...  does someone have a reliable list of what is and is not working or should I look at the code myself to get this documented correctly?

Reply:
@mosabua I think no one has such list and we need to look at the code. As you already know, "Override Methods.." in IntelliJ provides us missing statement types in each connector.

Reply:
I updated the list a bit based on some poking around ..